src/sagas/repository.ts

```diff
@@ -362,6 +362,29 @@ export function* removeSnapshotPolicyMember({ payload }: any): any {
   }
 }
 
+export function* removeSnapshotPolicyMembers({ payload }: any): any {
+  const { snapshotId, users, policy } = payload;
+  const urls = users.map(
+    (user: string) =>
+      `/api/repository/v1/snapshots/${snapshotId}/policies/${policy}/members/${user}`,
+  );
+  try {
+    yield all(urls.map((url: string) => call(authDelete, url)));
+    yield put({
+      type: ActionTypes.REMOVE_SNAPSHOT_POLICY_MEMBERS_SUCCESS,
+    });
+    yield put({
+      type: ActionTypes.GET_SNAPSHOT_POLICY,
+      payload: snapshotId,
+    });
+  } catch (err) {
+    showNotification('An error occurred.  Please try removing the workspace again.');
```

Comment:
@okotsopoulos - this is the error text for the toast that appears whenever there's a problem removing one of the policy members.  Since the only item currently leveraging this is our new feature, the error is tailored to our use case.  In the future we could include tailored error messages in the payloads and default to generic responses if they're not provided.